sdks/java/io/cassandra/src/test/java/org/apache/beam/sdk/io/cassandra/CassandraIOIT.java

```diff
@@ -70,159 +65,166 @@
 @RunWith(JUnit4.class)
```

Comment:
Since this test uses the "--numberOfRecords" pipeline option, could you add it in the example above (just for clarity)?

Reply:
ok